config/routes.rb

```diff
@@ -11,4 +11,7 @@
   get "/account", to: "users#show"
   get "/login", to: "sessions#new"
   post "/login", to: "sessions#create"
+  delete "/logout", to: "sessions#destroy"
+
+  root to: "menu#index"
```

Comment:
You can leave the `to:` here. It can just be `root "menu#index"`. Is this where we want to send the user when they first visit the site, or do we want a welcome page? It's something we can do later, but if we are trying to recreate The Dirty Cookie, we should start thinking about mimicking parts of their design.